examples/visualization/plot_warp_surfaces.py

```diff
@@ -0,0 +1,83 @@
+"""
+=================================
+Warp head shapes between subjects
+=================================
+
+In this example, we warp data from one subject (sample) to another (fsaverage)
+using a spherical harmonic approximation of surfaces, followed by thin-plate
+spline (TPS) warping of the surface coordinates, as described in [1]_.
+
+References
+----------
+.. [1] Darvas F, Ermer JJ, Mosher JC, Leahy RM (2006). "Generic head models for
+       atlas-based EEG source analysis." Human Brain Mapping 27:129-143
+"""
+# Author: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+
+import mne
+
+fsaverage_path = op.join(op.dirname(mne.__file__), 'data', 'fsaverage')
+data_path = mne.datasets.sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+
+###############################################################################
+# Load the source digitization, destination surfaces, and source surfaces
+# (with everything in head coordinates):
+
+# Digitization
+info = mne.io.read_info(op.join(data_path, 'MEG', 'sample',
+                                'sample_audvis_raw.fif'))
+hsp = mne.bem.get_fitting_dig(info, ('cardinal', 'extra'))
+
+# Destination head surface
+fsaverage_surfs = [mne.read_bem_surfaces(op.join(fsaverage_path,
+                                                 'fsaverage-%s.fif' % kind))[0]
+                   for kind in ('head', 'inner_skull-bem')]
+fsaverage_trans = mne.read_trans(op.join(fsaverage_path,
+                                         'fsaverage-trans.fif'))
+for surf in fsaverage_surfs:
+    mne.surface.transform_surface_to(surf, 'head', fsaverage_trans)
```

Comment:
yeah it is abnormal. Worth the deprecation cycle to get to `copy=True` default or do you want to live with `copy=False` default (which I will add)